front/src/common/BootstrapSNCF/SelectImprovedSNCF.tsx

```diff
@@ -153,7 +155,7 @@ function SelectImproved<T extends string | SelectOptionObject>({
               </button>
             </div>
           </div>
-          <div className="select-menu" id="-selecttoggle">
+          <div className={`select-menu ${blockMenu ? 'position-relative' : ''}`} id="-selecttoggle">
```

Comment:
Import cx
```
import cx from 'classnames';
```

 and use it like this 
```suggestion
          <div className={cx('select-menu', { 'position-relative': blockMenu })} id="-selecttoggle">
```